wagtail/blocks/base.py

```diff
@@ -568,10 +568,7 @@ def render_with_errors(self, name, value, attrs=None, errors=None, renderer=None
 
```

Comment:
In the line above this, it looks like we will have to change the default error json to an empty object.

```py
            error_json = json.dumps({})
```

@gasman  - I can do some digging but do you know if `value_json` will usually be a List and `error_json` will usually be an ?? (List) or can they be any structure (inc `None`)?